cluster/gce/util.sh

```diff
@@ -2502,12 +2511,12 @@ function kube-up() {
   load-or-gen-kube-bearertoken
   load-or-gen-kube-clienttoken
 
-  # Make sure we have the tar files staged on Google Storage
-  find-release-tars
-  registry-authentication
-  create-and-upload-etcd-image
-  create-and-upload-etcd-empty-dir-cleanup-image
-  upload-tars
+  if [[ -z ${UPLOAD_TAR_DONE:-} ]]; then
+    echo "DBG: uploading image + tar files..."
+    test_resource_upload
+    echo "DBG: (at kube-up) image + tar files uploaded"
+    UPLOAD_TAR_DONE="done in kube-up"
```

Comment:
will this flag, UPLOAD_TAR_DONE, visible to other process that calls kube-up() ?

Reply:
yes each partition eventually calls into kube-up.sh; if it is set, partition starting won't have to repeat uploading.